src/Datadog.Trace.AspNet/AspNetScopeManager.cs

```diff
@@ -4,23 +4,32 @@
 
 namespace Datadog.Trace.AspNet
 {
-    internal class AspNetScopeManager : IScopeManager
+    /// <summary>
+    /// Manage the scope in a AspNet application.
+    /// </summary>
+    public class AspNetScopeManager : IScopeManager
```

Comment:
Any particular reason we made this `public`?

Reply:
See comment above for `SpanEventArgs` change